recipe/meta.yaml

```
@@ -6,39 +6,53 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/googlemaps-{{ version }}.tar.gz
-  sha256: 3055fcbb1aa262a9159b589b5e6af762b10e80634ae11c59495bd44867e47d88
+  url: https://github.com/googlemaps/google-maps-services-python/archive/refs/tags/v4.10.0.tar.gz
+  sha256: cc07f739731ba4e0a4586543ad14470695addff8f30840748b9c3b89e393aa36
+  patches:
+    - 0001-remove-superfluous-build-requirement.patch
 
 build:
-  noarch: python
+  skip: True                                            # [py<35]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
+  number: 0
+
 
 requirements:
+  build:
+    - patch                                             # [not win]
+    - m2-patch                                          # [win]
   host:
-    - python >=3.7
-    - requests >=2.20.0,<3.0
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.7
+    - python
     - requests >=2.20.0,<3.0
 
 test:
   imports:
     - googlemaps
   commands:
     - pip check
+    - pytest                                            # [not win]
+    - pytest --ignore="tests/test_timezone.py"          # [win]
```

```suggestion
    - pytest                                              # [not win]
    # test_los_angeles_with_no_timestamp raises OverflowErrorfails on win with OSError failure
    # because the timestamp value used 1680 is not between 1970 and 2038.
    - pytest -k "not test_los_angeles_with_no_timestamp"  # [win]
```

there are only two tests in the file so it is OK to ignore the entire file (avoiding setup/teardown), but you could also just skip the single test in this way. Also, we need a comment as to why we skip.

---

Thanks for the tip ! I agree that it is much clearer with the comments too.